queries.sql

```diff
@@ -0,0 +1,23 @@
+-- Find all animals whose name ends in "mon"
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 

> Take a screenshot of the results of your queries.
Remember to add the screenshot with the results of your queries to your Pull Request description.

- [ ] According to the project requirement, you were supposed to take screenshots of all these queries and include them in the pull request description, currently, I can you have a screenshot in a .png format, I'll suggest you also include the screenshot in the pull request description as required and for easy viewing. :+1: :smile:

Reply:
Hello @V-Blaze I've tried adding the screenshot in the PR and its not showing. Regarding the licence link I've updated it